data/json/items/ranged/slings.json

```
@@ -86,13 +85,13 @@
     "techniques": [ "WBLOCK_2", "RAPID", "SWEEP" ],
     "skill": "throw",
     "ammo": [ "rock" ],
-    "weight": "2000 g",
-    "volume": "3 L",
+    "weight": "1600 g",
+    "volume": "1650 ml",
     "longest_side": "180 cm",
     "price_postapoc": 250,
-    "to_hit": 1,
+    "to_hit": 2,
```

```suggestion
    "to_hit": 1,
```
Revert as per Erk's suggestion